src/include/sof/audio/mic_privacy_manager.h

```diff
@@ -0,0 +1,71 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2025 Intel Corporation. All rights reserved.
+ *
+ */
+
+#ifndef ADSP_FW_MIC_PRIVACY_MANAGER_H
+#define ADSP_FW_MIC_PRIVACY_MANAGER_H
+
+#include <zephyr/drivers/mic_privacy/intel/mic_privacy.h>
+#include <sof/lib/notifier.h>
+#include <ipc4/base-config.h>
+#include "../audio/copier/copier_gain.h"
+
+#define ADSP_RTC_FREQUENCY	32768
+
+struct mic_privacy_data {
+	enum ipc4_sampling_frequency audio_freq;
+	uint32_t mic_privacy_state;
+	bool dma_data_zeroing;
+	uint32_t fade_in_out_bytes;
+	uint32_t max_ramp_time_in_ms;
+
+	struct copier_gain_params mic_priv_gain_params;
+};
+
+struct mic_privacy_settings {
+	enum mic_privacy_policy mic_privacy_mode;
+	/* 0-Mic Unmute, 1-Mic Mute */
+	uint32_t mic_privacy_state;
+	uint32_t max_ramp_time;
+	union mic_privacy_mask privacy_mask_bits;
+};
+
+struct privacy_capabilities {
+	uint32_t privacy_version;
+	uint32_t capabilities_length;
+	uint32_t capabilities[1];
+};
+
+//describes gain direction
+enum mic_priv_gain_direction {
+	MIC_PRIV_ADDITION	= 0,
+	MIC_PRIV_SUBTRACTION	= 1,
+};
+
+enum mic_privacy_state {
+	MIC_PRIV_UNMUTED	= 0,
+	MIC_PRIV_FADE_IN	= 1,
+	MIC_PRIV_FADE_OUT	= 2,
+	MIC_PRIV_MUTED		= 3,
+};
```

Comment:
Do these values have to stay the same, or are they purely internal and can be easily changed between builds? If the latter, maybe you don't need explicit values

Reply:
Pure internal, removed